bots/pr/src/main/java/org/openjdk/skara/bots/pr/SponsorCommand.java

```diff
@@ -136,9 +136,9 @@ public void handle(PullRequestBot bot, PullRequest pr, CensusInstance censusInst
         }
     }
 
+    // This method only has one statement now, but it is kept intentionally to meet the change in the future.
```

Comment:
I don't think we need a comment for this. To me it still makes sense to keep this method.

Reply:
Fixed.